fineract-provider/src/main/java/org/apache/fineract/infrastructure/jobs/service/JobStarter.java

```diff
@@ -45,14 +50,19 @@ public class JobStarter {
     private final JobExplorer jobExplorer;
     private final JobLauncher jobLauncher;
     private final JobParameterRepository jobParameterRepository;
+    private final List<JobParameterProvider> jobParameterProviders;
+    private final JobNameService jobNameService;
 
-    public void run(Job job, ScheduledJobDetail scheduledJobDetail, FineractContext fineractContext)
-            throws JobInstanceAlreadyCompleteException, JobExecutionAlreadyRunningException, JobParametersInvalidException,
-            JobRestartException {
+    public void run(Job job, ScheduledJobDetail scheduledJobDetail, FineractContext fineractContext,
+            List<JobParameterDTO> jobParameterDTOList) throws JobInstanceAlreadyCompleteException, JobExecutionAlreadyRunningException,
```

Comment:
I think we can make this a Set instead of a List and make the JobParameterDTO to implement an equals and hashcode. What do you think?